global_vars.tf

```diff
@@ -170,3 +170,7 @@ variable "dev_mode" {
   description = "Flag to enabled DEV Mode defaulted to False"
   default="False"
 }
+
+variable "rsyslog_server_IP" {
```

Comment:
I think this should be rsyslog_server_ip (lowercase to match the other variables, more consistent then)